client/src/lib/result.js

```diff
@@ -135,9 +153,93 @@ export function resultsForView(results, format, forecastView) {
     prevResult = currentResult;
   }
 
+  const secondComplete =
+    resultsForView.length > 1 && isComplete(resultsForView[1].projectedAverage);
+  const fourthComplete =
+    resultsForView.length > 3 && isComplete(resultsForView[3].projectedAverage);
+  if (secondComplete) {
+    for (let i = 0; i < resultsForView.length; i++) {
+      var result = resultsForView[i];
+      if (isSkipped(result.projectedAverage)) {
+        break;
+      }
```

Comment:
I would make the same check we do in the prior loop, which is equivalent.

```suggestion
      if (currentResult.attempts.length === 0) {
        // From this point forward all results are empty, so we are done
        break;
      }
```